logfire/_internal/main.py

```diff
@@ -1262,7 +1265,13 @@ def instrument_sqlalchemy(self, **kwargs: Unpack[SQLAlchemyInstrumentKwargs]) ->
         from .integrations.sqlalchemy import instrument_sqlalchemy
 
         self._warn_if_not_initialized_for_instrumentation()
-        return instrument_sqlalchemy(**kwargs)
+        return instrument_sqlalchemy(
+            **{  # type: ignore
```

Comment:
This is the same logic as the other instrumentations, I just realized at this point that this approach was a bit easier and less tedious.